visualizer/parseTree.js

```diff
@@ -609,15 +609,15 @@
           visibleChoice: isVisibleChoice(node, parent)
         });
 
+        ohmEditor.parseTree.emit('create:traceElement', el, rootTrace, node);
```

Comment:
I just need to check the result to found out whether there is a next step node that will be hided by collapsing, and the result is coming from the listener of `create:traceElement`. Anyway, I moved it back, and let the `create:traceElement` listener check the collapse status instead.